examples/feature_convert/cadquery/cube.py

```
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
```

Ruff (0.8.2)

Shebang is present but file is not executable

(EXE001)